darwin/dataset/download_manager.py

```diff
@@ -525,3 +554,119 @@ def _rg16_to_grayscale(path):
 
     new_image = Image.fromarray(np.uint8(image_2d_gray), mode="L")
     new_image.save(path)
+
+
+def _download_and_extract_video_segment(url: str, api_key: str, path: Path, manifest: dt.SegmentManifest) -> None:
+    _download_video_segment_file(url, api_key, path)
+    _extract_frames_from_segment(path, manifest)
+    path.unlink()
+
+
+def _extract_frames_from_segment(path: Path, manifest: dt.SegmentManifest) -> None:
+    try:
+        import cv2
```

Comment:
This dependency needs to be added to README with some explanation about . Right now, I managed to install it only by upping python version requirement and hardcoding `numpy` version. 

Also,
```
$ pip install "darwin[ocv]"
ERROR: Could not find a version that satisfies the requirement darwin[ocv] (from versions: none)
ERROR: No matching distribution found for darwin[ocv]
```

Reply:
Yeah the pip install message is only relevant after this PR gets deployed as it needs to be packaged for pip